nexus/types/src/external_api/params.rs

```diff
@@ -982,6 +982,32 @@ pub enum ExternalIpDetach {
     Floating { floating_ip: NameOrId },
 }
 
+/// A policy determining when an instance should be automatically restarted by
+/// the control plane.
+#[derive(Clone, Debug, Default, Deserialize, Serialize, JsonSchema)]
+#[serde(tag = "type", rename_all = "snake_case")]
```

Comment:
Uh...it's just because I copied the derives from another struct in this module, based on the (perhaps incorrect) assumption that we wanted to use that style for all enums. I'll change it back.